aspnetcore/fundamentals/openapi/samples/9.x/GetDocument.Insider/Program.cs

```diff
@@ -0,0 +1,16 @@
+using System.Reflection;
+
+var builder = WebApplication.CreateBuilder(args);
+
+if (Assembly.GetEntryAssembly()?.GetName().Name != "GetDocument.Insider")
+{
+   // 'IServiceCollection' does not contain a definition for 'AddDefaults' 
+   // builder.Services.AddDefaults();
+}
```

Comment:
@mikekistler  `AddDefaults` doesn't compile. Can you explain the naming conventions?
I don't understand how this code restricts code paths. I'd like the sample to restrict a config and a DB code path. I can add DB init code.